common/swagger/swagger.yaml

```diff
@@ -76,6 +76,11 @@ paths:
       produces:
         - application/json
       parameters:
+        - in: query
```

Comment:
hmmm why not just add optional arg to the like request? this seems like a strange extra to have

Reply:
I would think it better to add to `ScoreRequest` an extra parameter.

Reply:
like what would happen if the user didn't provide `?shapley_results=false` this shouldn't be a required parameter

Reply:
Its not a required parameter, so if it is not true or if the parameter is not passed/set then we would get just the prediction results

Reply:
ya... I think imho, it makes much more sense to include the parameter as part of the request `ScoringRequest` its weird that a majority of the parameterization of the request is done via `ScoringRequest` and there is just one parameter which is constructed into the url.

Reply:
I am +1 with @nkpng2k. Current "flags" are all in the req body. So I'd stay consistent.

Reply:
removed the query param and included in request object